09_json/kasunfdo/cmd/puppy-server/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/anz-bank/go-course/09_json/kasunfdo/pkg/puppy"
+	"github.com/anz-bank/go-course/09_json/kasunfdo/pkg/puppy/store"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	var dataFile *os.File
+	kingpin.Flag("data", "path to puppy data file").Short('d').Default("data.json").FileVar(&dataFile)
```

Comment:
I think if you want you could turn this into a oneliner with: 
`datafile := kingpin.Flag("data", "path to puppy data file").Short('d').Default("data.json").File()`
but maybe you purposefully chose not to do this to make it clearer that a file handle is returned?

Reply:
Yes. I think that way it would be clearer and easier :)